src/main/java/edu/umich/its/cpm/MigrationController.java

```diff
@@ -533,8 +533,14 @@ public Response getSiteMembershipAPI(
 			String sessionId = getUserSessionId(request);
 
 			// return CTools site members
-			return Response.status(Response.Status.OK).entity(get_site_members(siteId, sessionId))
-					.build();
+			HashMap<String, String> site_members = get_site_members(siteId, sessionId);
+			for (String userEid : site_members.keySet()) {
+				if (userEid.equals("errorMessage")) {
+						return Response.status(Response.Status.INTERNAL_SERVER_ERROR).entity(site_members).build();
+				}
+			}
+
+			return Response.status(Response.Status.OK).entity(site_members).build();
 		} catch (Exception e) {
```

Comment:
OK. Got your point. I logged it. 
How do i anticipate what exception might occur. If a code point don't have a obvious exception to throw i feel a generic exception can be used to catch instead of doing 
try{
//some code
}catch( RuntimeException || SocketException || RESTException){
log.error(e.getMessage())
}